bootstrapvz/providers/ec2/ebsvolume.py

```diff
@@ -4,16 +4,24 @@
 
 class EBSVolume(Volume):
 
-    def create(self, conn, zone):
-        self.fsm.create(connection=conn, zone=zone)
+    def create(self, conn, zone, encrypted, kms_key_id):
```

Comment:
You have defaults for the manifest of course, but you should probably add some defaults here as well in case somebody uses the code in a plugin to create a secondary ebs volume, for whatever reason.  
This would also make them named parameters, which would work better together with the EBS tag PR.